Neos.Media.Browser/Resources/Private/Translations/en/Main.xlf

```diff
@@ -427,6 +427,9 @@
 			<trans-unit id="missingDocumentNode" xml:space="preserve">
 				<source>No document node found for this node</source>
 			</trans-unit>
+			<trans-unit id="createMissingVariants" xml:space="preserve">
+				<source>Create missing Variants</source>
```

Comment:
variants should be in lowercase. Neos doesn't use title case in language labels as far as I know.